src/Dict.h

```diff
@@ -1162,11 +1189,13 @@ template <typename T> class Dictionary
 	void InsertRelocateAndAdjust(detail::DictEntry<T>& entry, int insert_position)
 		{
 /// e.distance is adjusted to be the one at insert_position.
-#ifdef ZEEK_DICT_DEBUG
+#ifdef DEBUG
```

Comment:
Is this change intentional? All of the other uses of ZEEK_DICT_DEBUG stayed the same.

Reply:
Noted, thanks.